### PR TITLE
[29]  Styling footer 

### LINKED ISSUE #29: Styling footer
- compare with an example in Figma
- add 'Code for Poznań' logo -> logo for the dark background (svg) - put it on the left side of the footer
- put smaller social-media logos on the right side of the footer
- delete 'Pomóż Alince' 
- delete 'O Nas'
- logo CfP should direct to CfP website

### PR DESCRIPTION
**Pull request by martabakaj**
Resolves #29

### CODE REVIEW

File: assets/scss/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.footer{
+    background-image: url("../images/footerbg.svg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.2rem 3rem 1.7rem;
+
+    .icons {
```
Comment (agatakozinska) [inline assets/scss/_footer.scss]: you don't need to nest icons and social medial inside footer, they can be a block (**B**EM) themself :)

File: assets/scss/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.footer{
+    background-image: url("../images/footerbg.svg");
+    background-size: cover;
+    background-repeat: no-repeat;
```
Comment (agatakozinska) [inline assets/scss/_footer.scss]: you can shorten it to one line: `background: url("../images/footerbg.svg") cover no-repeat;`

https://developer.mozilla.org/en-US/docs/Web/CSS/background

File: index.html [hunk 3]
```diff
@@ -113,12 +113,12 @@ <h3 class="titles download__title">Zacznij pracować z Alinką!</h3>
...
-      <div class="icons footer__icons">
-        <img
-          src="assets/images/Code_for_Poznan_final-01.png"
-          alt="Code for Poznan logo"
-          class="icons__logo"
-        />
+            <img
+            src="assets/images/Code_for_Poznan_final-01.png"
+            alt="Code for Poznan logo"
+            class="icons__logo"
```
Comment (agatakozinska) [inline index.html]: something here is wrong, when you are using BEM, first you should have block, in this case, `icons` then you can have elements of this block like `logo`, so version before with `div class="icons'` was ok :)

File: assets/scss/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    }
+    .social-media{
+        text-align: right;
+        
+        &__item{
+            margin-left: 1rem;
+            text-decoration: none;
+        }
+    }
+
```
Comment (agatakozinska) [inline assets/scss/_footer.scss]: unnecessary space

**Review by agatakozinska — CHANGES_REQUESTED** [review]

File: assets/scss/_footer.scss [hunk 5]
```diff
@@ -1,32 +1,24 @@
 .footer{
-    background-image: url("../images/footerbg.svg");
-    background-size: cover;
-    background-repeat: no-repeat;
+    background-image: url("../images/footerbg.svg") cover no-repeat;
```
Comment (agatakozinska) [inline assets/scss/_footer.scss]: only background can take all the properties, so please use it instead `background-image`

**Review by agatakozinska — CHANGES_REQUESTED** [review]
as I told you please remove all lines left after merging in index.html

File: assets/scss/_footer.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
+.footer{
+    background-image: url("../images/footerbg.svg") cover no-repeat;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.2rem 3rem 1.7rem;}
```
Comment (Bosmanfrx) [inline assets/scss/_footer.scss]: newline before brace

File: index.html [hunk 7]
```diff
@@ -113,12 +113,28 @@ <h3 class="titles download__title">Zacznij pracować z Alinką!</h3>
       />
     </section>
     <footer class="footer">
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> ed726cabe1608c760dd59cc7767db741c9a1f5fa
```
Comment (Bosmanfrx) [inline index.html]: some rogue conflicts ;)

**Review by Bosmanfrx — DISMISSED** [review]

File: index.html (hunk 7 above)
Comment (martabakaj) [reply index.html]: resolved

File: assets/scss/_footer.scss (hunk 6 above)
Comment (martabakaj) [reply assets/scss/_footer.scss]: done

File: assets/scss/_footer.scss (hunk 5 above)
Comment (martabakaj) [reply assets/scss/_footer.scss]: done

File: assets/scss/_footer.scss (hunk 4 above)
Comment (martabakaj) [reply assets/scss/_footer.scss]: done

File: index.html (hunk 3 above)
Comment (martabakaj) [reply index.html]: for me logo is not an icon so I've changed it into footer__logo

File: assets/scss/_footer.scss (hunk 2 above)
Comment (martabakaj) [reply assets/scss/_footer.scss]: done

File: assets/scss/_footer.scss (hunk 1 above)
Comment (martabakaj) [reply assets/scss/_footer.scss]: changed

File: .travis.yml [hunk 8]
```diff
@@ -0,0 +1,24 @@
+language: node_js
```
Comment (magul) [inline .travis.yml]: this file should not be in pull request diff. It looks like problem with backmerging.

File: assets/images/description-bg.svg [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<svg
```
Comment (magul) [inline assets/images/description-bg.svg]: ditto

File: assets/images/intro-graphics.svg [hunk 10]
```diff
@@ -0,0 +1,105 @@
+<svg width="775" height="564" viewBox="0 0 775 564" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (magul) [inline assets/images/intro-graphics.svg]: ditto

File: assets/scss/_footer.scss (hunk 6 above)
Comment (magul) [reply assets/scss/_footer.scss]: That's still not resolved @martabakaj

File: assets/scss/_icons.scss [hunk 11]
```diff
@@ -0,0 +1,8 @@
+.icons {
+    &__email{
+        color: $white;
+        font-family: Montserrat, sans-serif;
+        font-weight: bold;
+        margin-bottom: 1.2rem;
+    }
+}
```
Comment (magul) [inline assets/scss/_icons.scss]: newline please

File: package.json [hunk 12]
```diff
@@ -1,5 +1,7 @@
 {
   "name": "alinka-website",
+  "homepage": "https://alinka.io",
```
Comment (magul) [inline package.json]: These changes were probably introduced in different pull request previously and this diff is a result of wrong back merge.

**Review by magul — DISMISSED** [review]
There are few issues, that need to be resolved before even starting proper code review. We can sit down and resolve them next Wednesday.

**Review by agatakozinska — CHANGES_REQUESTED** [review]
file footer.scss doesn't look fine, please fix an indentation and is a style for socialmedia really ok? you don't have much there

File: assets/scss/_footer.scss [hunk 13]
```diff
@@ -0,0 +1,16 @@
+.footer {
+    background: url("../images/footerbg.svg") center/cover no-repeat;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```
Comment (agatakozinska) [inline assets/scss/_footer.scss]: hmmm I looked at it and something is wrong on mobile is shows also horizontal scroll which is not good, when you remove flex it seems it's working fine
I guess you wanted to have all children centered, for the img you can use freely text-aling (it's not only for texts) and use flex just for `.footer__content`:) 
if you want to have a control for all the things inside footer you can just wrap them in another div (not that one with background with svg)
Screenshot from 2020-04-01 00-25-03

File: assets/scss/_icons.scss [hunk 14]
```diff
@@ -0,0 +1,8 @@
+.icons {
```
Comment (agatakozinska) [inline assets/scss/_icons.scss:1]: icons are too small

**Review by agatakozinska — CHANGES_REQUESTED** [review]

File: index.html [hunk 15]
```diff
@@ -1,106 +1,104 @@
...
+          <img src="assets/images/github.svg" alt="Github-link" />
+        </a>
       </div>
-    </footer>
-  </body>
+    </div>
+
+  </footer>
+</body>
+
```
Comment (agatakozinska) [inline index.html]: remove unneeded spaces between tags in a whole file

**Review by agatakozinska — CHANGES_REQUESTED** [review]

File: index.html (hunk 15 above)
Comment (martabakaj) [reply index.html]: done

**Review by agatakozinska — APPROVED** [review]

**Pull request merged**